src/openrct2/localisation/Language.h

```diff
@@ -17,10 +17,7 @@
 #ifndef _LANGUAGE_H_
 #define _LANGUAGE_H_
 
-#ifdef __cplusplus
 #include <string>
```

Comment:
This line seems to be causing trouble for XCode. The file is still included in `macos.m`, which is an Objective-C file.